.github/workflows/linter.yml

```
@@ -0,0 +1,23 @@
+# Run eslint on the codebase
+name: Lint codebase
+
+on:
+  pull_request:
+    branches: '**'
+
+jobs:
+  lint:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
+    - uses: awalsh128/cache-apt-pkgs-action@latest
+      with:
+        packages: make npm
+        version: 1.0
+    - name: Install eslint
+      run: |
+        sudo npm install -g eslint eslint-plugin-jsdoc
```

Isn't it working without sudo ?

```suggestion
        npm install -g eslint eslint-plugin-jsdoc
```

---

Ah, that command failed without sudo on my machine, I didn't test within the workflow